### ISSUE TITLE
[Launch checklist] TN Workflows tool: Compliant Reporting

### ISSUE THREAD

**Issue by sychang** [Recidiviz/pulse-dashboard#1714]
**Target timeline**
- 3/30: Preview to District Directors
- 3/31: Rollout to D50 TTs (~10) --> Test onboarding can proceed without this checklist being completed because we are manually walking everyone onboard.
- Week of 4/11: Full state rollout --> This launch checklist is blocking the full-state rollout

**Relevant links**
- PRD: https://docs.google.com/document/d/1fE13vIHgt9vmhspKSPG7OiXV2-QNutNRJVV0LXP29XY/edit#heading=h.x0hh13pdmi33
- Engineering Design Docs: https://paper.dropbox.com/doc/Design-Doc-Practices-V2-Compliant-Reporting--BeMLJB0OmVG36gBWe~sq_zKkAg-8tLyk6uTpkuSfNRKUAxPM
- Mocks or staging: https://www.figma.com/file/pkhIrK2QaZi37rDpV5slBK/Practices-v2?node-id=2116%3A676
- End user documentation and comms doc: Go-to-market doc: https://docs.google.com/document/d/1UihGqjQRWZIAPjVUHdfKs5IMRL3vQTNWyN7_lq6FKZI/edit#
- Privacy design doc: https://docs.google.com/document/d/1iYGNusqPKnhG6sn4O6TgySS_bi4B4a9ybdYjTq3xlNg/edit?usp=sharing
- Change management schedule: Refer to GTM doc
- Analysis plan: https://docs.google.com/document/d/1CZXr8wN7s-w8hUWGvgy6LMgGj0dg0DmYOmPqMzTTeck/edit#

**What size change is being launched?**
Large - this is a new tool that will be rolling out to all supervision staff in a new state

**Please link the security review bug:**
Spreadsheet here: https://docs.google.com/spreadsheets/d/1OJSSLDppb7NSc_de2mavNpSs2Dmi6xk9tRNvtHC4OCA/edit#gid=0

**Which states or clients is this change going to? Which non-state organizations will have access?**
TDOC staff will have access to this.

**Write a short description of the change:**
We are rolling out the initial version of a tool that will enable TDOC staff to transfer 1000+ people to compliant reporting over the course of April.

**Does this change collect or show any new PII (of state staff, individuals under supervision or in facilities, etc.)?** 

_If new PII or PII-handling, privacy review is necessary._

Yes - there will now be a list of individuals shown directly in the tool. See PDD for more.

**Will this change require change management (managing end-users from an existing workflow to a new workflow)?**

Yes, refer to [GTM doc](https://docs.google.com/document/d/1UihGqjQRWZIAPjVUHdfKs5IMRL3vQTNWyN7_lq6FKZI/edit#)

**Launch approval checklist. Check box and add date when sign-off is obtained**
- [x] Exec: @clementinej @jessex [03/25/2022]
- [x] Product: @cawarren
- [x] Eng: @Catacola  [03/30/2022]
- [x] Data: @DSharm 
- [x] Research: @zagoodman [03/30/2022]
- [x] Security: @phenggeler  [03/31/2022]
- [x] Compliance: @phenggeler  [03/31/2022]
- [x] Comms: @evangreenlowe   
- [x] Docs: @evangreenlowe   
- [x] Design: @agronomic    
- [x] Legal: @cawarren
- [x] Privacy: @cawarren 
- [ ] Accessibility: @agronomic   
- N/A Localization: Usually N/A

cc @macfarlandian @danawillow @ohaibbq @caroletouma @gheidkamp11

–––––––––––––––––– GLOSSARY ––––––––––––––––––

**Launch size**: A launch’s size is a rough estimate of its ‘splash’ - with users, with the press, etc. For example: 
- Large launch examples: New product offering or tool, major redesign or re-launch of existing product or tool.
- Medium launch examples: Launching an existing product or tool to a new state, adding a significant new feature in existing product or tool, updating the design in a noticeable way for an existing feature or set of features, 
- Minor launch examples: Small feature additions or changes to existing products or tools, improvements to methodology or ease of use. Generally not noticeable to the majority of end-users.

**Comment by jessex** [comment]
Checking the exec bit for acknowledgement of this very big launch!

Will be following along on subsequent bits and am happy to jump in anywhere I can be of help.

**Comment by DSharm** [comment]
Checking the data bit! The following are outstanding things that are worth noting but not launch blocking. CC @gheidkamp11 to jump in if I'm wrong!

- The process for keeping data fresh is a bit manual (copying from prod to staging on Mondays and Thursdays, and uploading standards spreadsheet using admin panel on the same days), but it should work, not be a huge lift, or be launch blocking
- `AssignedStaff` has encoding issues which may make certain fields (e.g. `supervision_start_date`) stale. The fields depending on that data don't directly determine CR eligibility so some staleness there should not be launch-blocking, but should be resolved hopefully within this week
- There are known open questions about the CR logic that may reveal bugs or result in changes once we begin surfacing data in front of TTs (these have been noted in the GTM plan)

**Comment by Catacola** [comment]
Engineering looks good! We've bug-bashed 2.5 times and had an external tester validate the flows.

**Comment by evangreenlowe** [comment]
Feel good about comms and documentation for tomorrow - plenty enough to guide TTs through tomorrow's sessions

**Comment by agronomic** [comment]
Just want to flag that there are some accessibility issues that should not block the release and we should do a fast-follow update with fixes. Thoughts/concerns?

**Comment by sychang** [comment]
Thanks for flagging, @agronomic ! Agree with not blocking this rollout to <10 TTs since we'll be manually walking them through. Let's note the critical accessibility fixes to prioritize before the 4/11 rollout.

**Comment by zagoodman** [comment]
Research is on board with the rollout plan! We clarified today that the statewide rollout (week of 4/11) will include flagging clients eligible for compliant reporting _both_ with and without PO discretion required. We'll analyze the key metric of 'days eligible for CR' disaggregated by whether discretion is required.

I added a link to the analysis plan in the description above. Power calcs are WIP and will be complete and registered before 4/11.
